read me update (#1290)
CMSgov/cms-mdct-qmr: README.md

- npm test

> ## Destroy

Reviewer: Maybe change to "Local Destroy Instructions"

> Want to destroy from your Mac? This may be necessary in some cases where you have orphaned infrastructure

Reviewer: Not everyone will be on a mac

> - Install/configure the AWS CLI (You will have to copy AWS keys into your terminal)

Reviewer: I'd expand this out to talk about the steps to retrieve those credentials from the cloudtamer site

> - brew install jq (if you don't have already installed)

Reviewer: What is jq? why are we installing it (some brief description will help give context)

> - sh destroy.sh "name_of_your_branch" and re-enter the branch name once it has found all associated resources

Reviewer: Are there any troubleshooting things to try if this command fails?

- npm test

> ## Local Destroy Instructions

Reviewer 2: Can we change this slightly? To me "Local Destroy Instructions" sounds like it refers to something in my local environment. 

I don't have a perfect alternative, but maybe something like this?
- Destroy Orphaned AWS Infrastructure
- Destroy Infrastructure from Command Line

Want to destroy locally? This may be necessary in some cases where you have orphaned infrastructure

- Install/configure the AWS CLI (You will have to copy AWS keys into your terminal)
- Go to https://cloudtamer.cms.gov/portal/project and login with your EUA

> - Go to the mdct-qmr-dev project and click on "Cloud access", click on your account, click on mdctqmr-developer-admin, click on Short-term Access Keys, Then paste the the text from Option 1 into your terminal

Reviewer 2: I think these could be individual bullets

> - brew install jq (if you don't have already installed) (jq is command-line JSON processor necessary for the destroy script to run properly)

Reviewer 2: ```suggestion
- `brew install jq` Install jq (command-line JSON processor). This is necessary for the destroy script to run properly.
```

> - sh destroy.sh "name_of_your_branch" and re-enter the branch name once it has found all associated resources. (There shouldn't be any errors but if there are any. Re-running the script should fix it)

Reviewer 2: ```suggestion
- `sh destroy.sh name_of_your_branch` Run destroy script. You will be prompted to re-enter the branch name once it has found all associated resources. (There shouldn't be any errors but if there are any. Re-running the script should fix it)
```
